fem/bilininteg.hpp

```diff
@@ -1835,6 +1835,11 @@ class VectorMassIntegrator: public BilinearFormIntegrator
    Coefficient *Q;
    VectorCoefficient *VQ;
    MatrixCoefficient *MQ;
+   // PA extension
```

Comment:
May be out of scope for this PR, but I am wondering if the next four lines (which are copied verbatim in at least four places) should be in the base class? (Or maybe in a separate PA base class, with multiple inheritance, which is probably a mistake.) I just don't like seeing these member data rewritten everytime we implement a PA `BilinearForm`.